.github/workflows/smoke-test.yml

```diff
@@ -37,13 +49,13 @@ jobs:
         run: cargo fmt --manifest-path=Cargo.toml --all --check
 
       - name: Build
-        run: cargo build --verbose --features untrusted,bindings
+        run: cargo build --verbose --features untrusted,bindings,polars
```

Comment:
Background question: What is gained by separating the polars feature? Is it for Rust users who might want only a subset of functionality? Is it to keep developers from accidentally adding dependencies to libraries where they aren't really needed?

We have some cleanup to do on the existing features (#1343, #1338), so I'd like to understand what we gain.

Reply:
1. R will not support Polars out-of-the-box, so R users don't need to compile it (it's a big dep)
2. Users can disable Polars and get a WebAssembly build once OpenSSL is dropped
3. We were thinking of initially making this part of a patch release, where it would be better to leave it off by default. However, there are some other breaking changes in main, so this is not applicable.